### PR TITLE
Add to gql query to retrieve graph names off of repository

### PR DESCRIPTION
**Pull request by dpeng817**
Not sure how to test this, but hoping to use this to unblock dish for getting an accurate list of graphs in dagit.

### CODE REVIEW

File: js_modules/dagit/packages/core/src/workspace/WorkspaceContext.tsx [hunk 1]
```diff
@@ -62,6 +62,7 @@ const ROOT_REPOSITORIES_QUERY = gql`
               repositories {
                 id
                 name
+                graph_name_list
```
Comment (hellendag) [inline js_modules/dagit/packages/core/src/workspace/WorkspaceContext.tsx:65]: We should stick to `camelCase` on these.

File: js_modules/dagit/packages/core/src/workspace/WorkspaceContext.tsx [hunk 2]
```diff
@@ -1,13 +1,13 @@
-import {ApolloQueryResult, gql, useQuery} from '@apollo/client';
+import { ApolloQueryResult, gql, useQuery } from '@apollo/client';
```
Comment (hellendag) [inline js_modules/dagit/packages/core/src/workspace/WorkspaceContext.tsx:1]: Hmm, I think you might have Prettier misconfigured somehow.

**Review by hellendag — COMMENTED** [review]
We will also want to be able to load graphs by name, e.g. navigating to `/workspace/repo@location/graph/foobar`.

**Comment by alangenfeld** [comment]
I don't think returning the list of names is an improvement since `graphName` is already available on `Pipeline` . The only lift would be actually making a `Graph` type and a `graphs` field on `repository` that returns them all.

> We will also want to be able to load graphs by name, e.g. navigating to /workspace/repo@location/graph/foobar.

How do we do this lookup for pipelines currently. Is it not via the `WorkspaceContext` ?

**Review by prha — DISMISSED** [review]
+1 alex... let's just add `graphName` to the pipeline section of the workspace context query?

**Review by prha — APPROVED** [review]

**Pull request merged**